src/TrackPanelResizeHandle.h

```diff
@@ -58,9 +58,9 @@ class TrackPanelResizeHandle final : public UIHandle
 
    bool mInitialMinimized{};
    int mInitialTrackHeight{};
-   int mInitialActualHeight{};
+   int mInitialExpandedHeight{};
    int mInitialUpperTrackHeight{};
-   int mInitialUpperActualHeight{};
+   int mInitialUpperExpandedHeight{};
```

Comment:
While working on resizers, i noticed that this class can be split into two: TrackResizeHandle (which could be used to resize tracks) and TrackChannelResizeHandle (could be used as a resizer between channels). What do you think?

Reply:
A good idea that might make things plainer.

I consider the BackgroundCell and the TrackPanelResizerCell and its associated handle to be close collaborators of TrackPanel, which also defines the sub-division of the track area that uses those cells.

But I want TrackInfo and the sub-classes of TrackView not to have strong assumptions built into them about how TrackPanel subdivides its area.  They are meant just to supply methods for cells, but not to decide where those cells are and how big.

This division might make it easier to program alternative TrackPanel arrangements if we ever do want that.

This PR achieves this for vertical divisions -- only one constant kVerticalPadding is widely known among the other cell classes.  It's not yet so encapsulated for the horizontal axis -- there is still mention of left margin and right margin in other places.  But the last commits attempted a few things for that too.